src/widgets/Tabs/PatientSelect.js

```diff
@@ -49,89 +55,50 @@ import { generateUUID } from '../../database/index';
 const PatientSelectComponent = ({
   createPatient,
   formConfig,
-  isAscending,
   onCancelPrescription,
   onFilterData,
-  onSortData,
   patients,
   selectPatient,
-  sortKey,
 }) => {
   const columns = React.useMemo(() => getColumns(MODALS.PATIENT_LOOKUP), []);
   const { pageTopViewContainer } = globalStyles;
-
-  const renderRow = React.useCallback(
-    listItem => {
-      const { item, index } = listItem;
-      const rowKey = item.id;
-      return (
-        <DataTableRow
-          rowData={patients[index]}
-          rowKey={rowKey}
-          getCallback={() => selectPatient(item)}
-          columns={columns}
-          rowIndex={index}
-          onPress={() => selectPatient(item)}
-        />
-      );
-    },
-    [patients]
-  );
-
-  const renderHeader = React.useCallback(
-    () => (
-      <DataTableHeaderRow
-        columns={columns}
-        isAscending={isAscending}
-        sortKey={sortKey}
-        onPress={onSortData}
-      />
-    ),
-    [columns, sortKey, isAscending]
-  );
+  const keyboardIsOpen = useKeyboardIsOpen();
```

Comment:
this is for making the bottom button row dissapear when the keyboard is opening. It's a little janky though, might need to remove